tools/buildsys/src/manifest.rs

```diff
@@ -34,6 +34,24 @@ sha512 = "123456"
 [package.metadata.build-variant]
 included-packages = ["release"]
 ```
+
+`package-name` lets you override the package name in Cargo.toml; this is useful
+if you have a package with "." in its name, for example, which Cargo doesn't
+allow.  This means the directory name and spec file name can use your preferred
+naming.
+```
+[package.metadata.build-package]
+package-name = "better.name"
+```
+
+`variant-sensitive` lets you specify whether the package should be rebuilt when
+building a new variant, and defaults to false; set it to true if a package is
+using the variant to affect its build process.  (Typically this means that it
+reads BUILDSYS_VARIANT.)
+```
+[package.metadata.build-package]
+variant-sensitive = true
+```
```

Comment:
nit: maybe order these with the other `build-package` values, before the `build-variant` ones.

Reply:
I've reorganized these and added section headers to make the difference clearer.